src/commands/bundles.js

```diff
@@ -0,0 +1,160 @@
+const { toSnakeCase, tryRequire, writeFile, stripWhitespace } = require('../utils');
+const { authorize, sheets } = require('../sheets');
+
+const generateLocaleExports = (adapter, outputPath, locales) => {
+  const { index, indexName } = adapter;
+  const localeExportPath = `${outputPath}/${indexName}`;
+  const existingLocales = tryRequire(localeExportPath) || false;
+
+  if (existingLocales) {
+    const existingLocaleKeys = Object.keys(existingLocales).sort();
+
+    if (
+      locales.length === existingLocaleKeys.length &&
+      locales.sort().every((loc, idx) => loc === existingLocaleKeys[idx])
+    ) return;
+  }
+
+  const localeList = locales.map((locale) => [toSnakeCase(locale).toUpperCase(), locale]);
+  const i18nIndexFile = stripWhitespace(index({ list: localeList }));
+
+  return writeFile(localeExportPath, i18nIndexFile);
+}
+
+const generateBundleExports = (adapter, outputPath, files, locales) => {
+  const { index, indexName } = adapter;
+  if (!files || !files.length || files.every(f => !f.newFile)) return;
+
+  const bundleList = (files || []).map((file) => (
+    [file.bundleName, file.fileName]
+  ));
+
+  bundleList.sort((array) => array[0]);
+
+  const langIndexFile = stripWhitespace(index({ list: bundleList, mirrorExports: true }));
```

Comment:
wondering if we should opt for a more descriptive method name over `index`. it makes sense in the adapter definition but it's hard to tell what it's doing out of context.  maybe  `generateIndex` or `createExportTemplate`?

also, how about running stripWhitespace in `index` if stripping the return value is consistent like it is now